wagtailmodelchoosers/client/components/BaseChooser.js

```diff
@@ -20,14 +20,9 @@ const defaultProps = {
 const propTypes = {
   initialValue: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
   updateInputValue: PropTypes.func.isRequired,
-  label: PropTypes.string.isRequired,
   initial_display_value: PropTypes.string.isRequired,
-  endpoint: PropTypes.string.isRequired,
-  value: PropTypes.any,
   required: PropTypes.bool.isRequired,
   display: PropTypes.oneOfType([PropTypes.string, PropTypes.array]).isRequired,
-  list_display: PropTypes.array.isRequired,
-  filters: PropTypes.array,
```

Comment:
I think I can agree with decomposing `...this.props`. I think it was handy in development because things were changing but this can go away now.